x-pack/plugins/ingest_manager/server/services/config.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Observable, Subscription } from 'rxjs';
+import { IngestManagerConfigType } from '../../common';
+
+/**
+ * Kibana config observable service, *NOT* agent config
+ */
+class ConfigService {
+  private observable: Observable<IngestManagerConfigType> | null = null;
+  private subscription: Subscription | null = null;
+  private config: IngestManagerConfigType | null = null;
+
+  private updateInformation(config: IngestManagerConfigType) {
```

Comment:
Similar to my comment on licensing, this could probably be replaced by using a BehaviorSubject